issue4/styles.css

```diff
@@ -0,0 +1,312 @@
+
+
+
+/* Projects page styles */
+
+.grid-item-project {
+    margin: 1em;
+    padding: 0.5em;
+    color: white;
+    background-color: rgba(109, 103, 93, 100%);
+    border: solid 1px black;
+    border-radius: 5px;
+}
+
+.project-img-container {
+    display: flex;
+    width: 100%;
+    height: 250px;
+    background-color: white;
+    color: black;
+    border: 1px solid black;
+    z-index: 0;
+}
+
+/* About page styles */
+
+.profile-img-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0.5em;
+    width: 12em;
+    height: 12em;
+}
+
+.slideshow-container {
+    width: 12em;
+    height: 12em;
+}
+
+/* Contact page styles */
+
+.contact-heading {
+    margin: 1em;
+    color: white;
+}
+
+.location-map {
+    margin: 1em;
+    height: 12em;
+    border: 1px solid black;
+    border-radius: 10px;
+}
+
+.grid-container--bg-img-mobile {
+    background-image: url("/images/cockpit-instruments-banner.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+@keyframes pageDisplay {
+    0% {opacity: 0;}
+    100% {opacity: 1;}
+}
+
+/* Media Queries*/
+@media (min-width: 481px) {
+    /* Site nav bar styles */
+    .nav-btn-container {
+        justify-content: flex-end;
+    }
+    
+    .grid-container {
+        grid-template-columns: repeat(8, 1fr);
+    }
+
+    .div--display-none {
+        display: none;
+    }
+
+    .grid-container--bg-img {
+        background-image: url("/images/cockpit-instruments-banner.jpg");
+        background-repeat: no-repeat;
+        background-size: cover;
+    }
+
+    /* Index page styles */
+
+   
+
+    /* Projects page styles */
+    .grid-item-project-1 {
+        grid-column: 1 / span 4;
+    }
+    
+    .grid-item-project-2 {
+        grid-column: 5 / span 4;
+    }
+    
+    .grid-item-project-3 {
+        grid-column: 3 / span 4;
+    }
+
+    /* About page styles */
+    .grid-item-profile {
+        grid-column: 1 / span 4;
+        flex-direction: column;
+        justify-content: flex-start;
+    }
+
+    .grid-item-about {
+        grid-column: 5 / span 4;
+    }
+
+    /* Contact page styles */
+    .grid-item--flex-row {
+        flex-direction: row;
+        justify-content: space-evenly;
+        grid-column: 1 / span 8;
+    }
+
+    .grid-item--col-8 {
+        grid-column: span 8;
+    }
+
+    .grid-container--row-2 {
+        grid-template-rows: repeat(2, 1fr);
+    }
+}
+
+@media (min-width: 768px) {
+    /* Site nav bar styles */
+    .nav-btn-container {
+        justify-content: flex-end;
+    }
+    
+    .grid-container {
+        grid-template-columns: repeat(12, 1fr);
+    }
+
+    .div--display-none {
+        display: none;
+    }
+
+    .grid-container--bg-img {
+        background-image: url("/images/cockpit-instruments-banner.jpg");
+        background-repeat: no-repeat;
+        background-size: cover;
+    }
+     
+    /* Index page styles */
+    .hero-container__img {
+        width: 80%;
```

Comment:
Out of curiosity, is there a reason why you wanted the hero image to be `width: 80%` for large screen sizes? 

I see you have this CSS class defined above as `width: 100%` for smaller screen sizes. Maybe consider keeping the width at `100%` and reducing the height if overall image size is a factor. This will keep the experience consistent regardless of the device/screen used.

Reply:
I went ahead and removed the 80% width for the index page hero image.